src/TestTrait.php

```diff
@@ -39,6 +39,18 @@ public static function erisSetupBeforeClass()
         }
     }
 
+    /**
+     * @return array 
+     */
+    public function getTestCaseAnnotations()
+    {
+        //from TestCase of PHPunit
+        return \PHPUnit\Util\Test::parseTestMethodAnnotations(
```

Comment:
`\PHPUnit\Util\Test` was called `PHPUnit_Util_Test` in older releases.

You could check if the `getTestCaseAnnotations` method exists and call `\PHPUnit\Util\Test::parseTestMethodAnnotations` in case it doesn't.

Reply:
u mean check if the `getAnnotations` method from PHPUnit exists (in `$this`) ?

Reply:
yep, sorry, wrong copy paste